### PR TITLE
added logout event redirect to login

### LINKED ISSUE #23832: New admin console doesn't support automatic logout
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

admin/ui

### Describe the bug

Scenario like:
- Login to admin console https://localhost:8443/admin/master/console/ in tab1
- Open admin console also in another browser tab (tab2). I should be logged automatically due the SSO and it works this way.
- Click "Sign out" in tab2. I am logged-out, which is OK
- Go to tab1. I am still logged-in here, which is not OK. When I click to some link (For instance tab "Clients" on the left), I am redirected to login screen as SSO session was already removed, which is OK

### Version

22.0.4 (or latest Keycloak main)

### Expected behavior

Automatic logout in tab1 due the session iframe after user logged-out in tab2. 

### Actual behavior

Not being logged-out automatically in tab1 until I click on some link

### How to Reproduce?

See above

### Anything else?

Not 100% sure if this can be strictly considered a bug or missing functionlity (feature). On the other hand, this worked in the old admin console and hence I consider it as a bug and also as a regression, so adding regression label here.

### LINKED ISSUE #23833: Account console v2 doesn't support automatic logout
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

account/ui

### Describe the bug

Scenario like:

- Login to account console https://localhost:8443/admin/master/console/ in tab1. I did not enabled any preview features etc, so assume it is account console V2.
- Open account console also in another browser tab (tab2). I should be logged automatically due the SSO and it works this way.
- Click "Sign out" in tab2. I am logged-out, which is OK
- Go to tab1. I am still logged-in here, which is not OK. When I click to some link (For instance tab "Personal info" on the left), I am redirected to login screen as SSO session was already removed, which is OK

### Version

22.0.4 (or latest Keycloak main)

### Expected behavior

Automatic logout in tab1 due the session iframe after user logged-out in tab2.

### Actual behavior

Not being logged-out automatically in tab1 until I click on some link

### How to Reproduce?

See above

### Anything else?

Not 100% sure if this can be strictly considered a bug or missing functionality (feature). Also it is not a regression since old account console (Account console V1) was not based on javascript and automatic logout did not work here. So feel free to convert to "Feature" or "Enhancement".

### PR DESCRIPTION
**Pull request by edewit**
Closes #23832
Closes #23833

### CODE REVIEW

File: js/apps/admin-ui/src/keycloak.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export const keycloak = new Keycloak({
 });
 
 export async function initKeycloak() {
+  keycloak.onAuthLogout = () => keycloak.login();
```
Comment (jonkoops) [inline js/apps/admin-ui/src/keycloak.ts:12]: This callback can be applied directly after constructing the Keycloak instance, no need to wait until initialization.

**Review by jonkoops — CHANGES_REQUESTED** [review]
Let's also do this for the Account Console

File: js/apps/admin-ui/src/keycloak.ts (hunk 1 above)
Comment (edewit) [reply js/apps/admin-ui/src/keycloak.ts:12]: No because the return type of onAuthLogout is void

Comment (edewit) [reply js/apps/admin-ui/src/keycloak.ts:12]: Not according the documentation

Comment (jonkoops) [reply js/apps/admin-ui/src/keycloak.ts:12]: > The adapter supports setting callback listeners for certain events. Keep in mind that these have to be set before the call to the `init()` method.

So you can still set this outside of the `initKeycloak()` method, right after constructing the Keycloak instance.

**Review by jonkoops — APPROVED** [review]

**Pull request merged**